adswerve/tap-facebook
# capture Facebook API v9 changes from singer-io original branch

# Description of change
The original repo we forked from has made the cahnges required to the Facebook API v9.  Pulling these changes into our fork.

# Manual QA steps
 - run data transfer using the latest changes, check logs for errors.
 - verify data has been updated.
 
# Risks
 - low risk, Facebook did not make major changes to pulling marketing data with this version upgrade.
 
# Rollback steps
 - revert this branch
